plugins/modules/snmp_manager.py

```diff
@@ -330,10 +331,71 @@ def delete_snmp_manager(self, snmp_manager_id):
             LOG.error(msg)
             self.module.fail_json(msg=msg, **utils.failure_codes(e))
 
+    def validate_version_change(self, snmp_manager_params, snmp_manager_details):
```

Comment:
Does the API not handle this?

Reply:
This is done to handle error messages better.